docs/dev/README.md

```diff
@@ -0,0 +1,102 @@
+487e6c1bdb9a  localhost/podman-pause:4.9.3-1708357294                                          4 minutes ago  Up 3 minutes  0.0.0.0:8080->8080/tcp, 0.0.0.0:8090->8090/tcp, 0.0.0.0:8888->8888/tcp  7c18ebd0915a-infra
+8479b5eb8a8d  quay.io/sclorg/postgresql-12-c8s:latest                    run-postgresql        4 minutes ago  Up 3 minutes  0.0.0.0:8080->8080/tcp, 0.0.0.0:8090->8090/tcp, 0.0.0.0:8888->8888/tcp  assisted-installer-db
+ffb9013c4fab  quay.io/edge-infrastructure/assisted-installer-ui:latest   /deploy/start.sh      3 minutes ago  Up 3 minutes  0.0.0.0:8080->8080/tcp, 0.0.0.0:8090->8090/tcp, 0.0.0.0:8888->8888/tcp  assisted-installer-ui
+100c865abfd6  quay.io/edge-infrastructure/assisted-image-service:latest  /assisted-image-s...  3 minutes ago  Up 3 minutes  0.0.0.0:8080->8080/tcp, 0.0.0.0:8090->8090/tcp, 0.0.0.0:8888->8888/tcp  assisted-installer-image-service
+78924b68f7af  quay.io/edge-infrastructure/assisted-service:latest        /assisted-service     3 minutes ago  Up 3 minutes  0.0.0.0:8080->8080/tcp, 0.0.0.0:8090->8090/tcp, 0.0.0.0:8888->8888/tcp  assisted-installer-service
+```
+For testing we can use `curl`:
+```shell
+$ curl -i http://127.0.0.1:8090/api/assisted-install/v2/infra-envs
+HTTP/1.1 200 OK
+Content-Type: application/json
+Vary: Accept-Encoding
+Date: Wed, 11 Sep 2024 14:16:43 GMT
+Content-Length: 3
+
+[]
+
+$ curl -i http://127.0.0.1:8090/api/assisted-install/v2/clusters
+HTTP/1.1 200 OK
+Content-Type: application/json
+Vary: Accept-Encoding
+Date: Wed, 11 Sep 2024 14:18:59 GMT
+Content-Length: 3
+
+[]
+```
+A more detailed description of how to use the REST API is available in the [user guide](docs/user-guide/rest-api-getting-started.md).
+
+## assisted-test-infra
+[Detailed documentation](https://github.com/openshift/assisted-test-infra/blob/master/README.md)
+
+This scenario is useful for developing and testing our end-to-end tests.
+This environment requires a large amount of resources since it will deploy additional baremetal hosts (VMs), so **running it in local environments is not recommended, do it on a beaker box or an ephemeral server**.
+
+This environment requires a prior setup on the machine where it will run. The steps for installation are described in [this guide](https://github.com/openshift/assisted-test-infra/blob/master/GUIDE.md).
+
+## CRC
+[Detailed documentation](http://crc.dev/crc/getting_started/getting_started/introducing/)
+
+We can install a local environment (Openshift / OKD / Microshift) with the **crc** tool. `crc` deploys a SNO cluster in our local environment and provides us a complete environment where we can perform our tests. By having all (or most) of Openshift components, our work here could go beyond the assisted-service functionalities and test its integration with other components.
+
+There is a section to perform this type of installation in the [Openshift console](https://console.redhat.com/openshift/create/local).
+
+## infrastructure operator
```

Comment:
I see this paragraph more as minikube/kind rather than infra operator.
We could deploy assisted with both infra operator or repo's manifests, and we'd achieve the same level of functionality

Reply:
Same as [#r1812796099](https://github.com/openshift/assisted-service/pull/6744/files/f96a58deb2203491d3b1cca80e205315b180e9b0#r1812796099), maybe I can rename the section to `kind` instead of `infrastructure operator`.